mediaplatform/tests/test_models.py

```diff
@@ -350,6 +351,7 @@ def setUp(self):
 
     def test_creation_no_fields(self):
         """A Channel object should *not* be creatable with no field values."""
+        # TODO this should fail?
```

Comment:
Not sure. I *think* ``clean()`` should fail, but I don't think Django stops you creating things if they are not disallowed by the DB.